NBISweden/urdr
# Visual feedback on number of hours logged

## Related issue(s) and PR(s)
This PR closes #156



## Type of change

 
## List of changes made

- the sum cell for each day will have a check icon if 8 hours are logged on the day
- it will have a warning icon if less or more hours are logged
- the weekly sum cell will have the check icon if 40 hours were logged
- it will have a warning icon if less or more than 40 hours were logged
- the icons are only displayed for weeks in the past. (For the current week, it needs to be at least Saturday for the icons to be displayed).

## Screenshot of the fix


## Testing

- Go to a future week: you shouldn't see any icons
- Go to the current week: if it's before Saturday you shouldn't see any icons
- Go to a past week: you should see the icons as described above. 
- Make a change in the hours of a time entry to meet the cases described above: As soon as you type, the icon should update together with the sum. (You don't need to save)

## Further comments


## Definition of Done checklist
